SPECS/xo-lite.spec

```diff
@@ -20,7 +20,10 @@ browser without having to deploy anything.
 %install
 install -d -m 755 %{buildroot}/opt/xensource/www
 cp -a * %{buildroot}/opt/xensource/www
+# We are removing unnecessary files.
 rm %{buildroot}/opt/xensource/www/LICENSE %{buildroot}/opt/xensource/www/CHANGELOG.md
+# The build.json file is a temporary file that appeared in August 2025. It should disappear eventually.
```

Comment:
I would have said "that appeared with version 0.14.0" but it's fine as you wrote.